aea/context/base.py

```diff
@@ -45,6 +46,8 @@ def __init__(
         decision_maker_message_queue: Queue,
         decision_maker_handler_context: SimpleNamespace,
         task_manager: TaskManager,
+        default_connection: Optional[PublicId],
+        default_routing: Dict[PublicId, PublicId],
```

Comment:
In the past the agent context was not used for connection related stuff...I am not sure this is a good idea adding it here. But worth discussing.

Reply:
we were doing it anyway, because `default_routing` is passes in the constructor of `AEA` as keyword, and in turn forwarded to the `AgentContext` constructor.